lk2nd/display/panel/generated/lk_panel_hx8394d_720p_video.h

```diff
@@ -0,0 +1,263 @@
+	{ sizeof(hx8394d_720p_video_on_cmd_19), hx8394d_720p_video_on_cmd_19, 200 },
+	{ sizeof(hx8394d_720p_video_on_cmd_20), hx8394d_720p_video_on_cmd_20, 0 },
+	{ sizeof(hx8394d_720p_video_on_cmd_21), hx8394d_720p_video_on_cmd_21, 10 },
+	{ sizeof(hx8394d_720p_video_on_cmd_22), hx8394d_720p_video_on_cmd_22, 0 },
+	{ sizeof(hx8394d_720p_video_on_cmd_23), hx8394d_720p_video_on_cmd_23, 5 },
+	{ sizeof(hx8394d_720p_video_on_cmd_24), hx8394d_720p_video_on_cmd_24, 0 },
+};
+
+static char hx8394d_720p_video_off_cmd_0[] = {
+	0x28, 0x00, 0x05, 0x80
+};
+static char hx8394d_720p_video_off_cmd_1[] = {
+	0x10, 0x00, 0x05, 0x80
+};
+
+static struct mipi_dsi_cmd hx8394d_720p_video_off_command[] = {
+	{ sizeof(hx8394d_720p_video_off_cmd_0), hx8394d_720p_video_off_cmd_0, 0 },
+	{ sizeof(hx8394d_720p_video_off_cmd_1), hx8394d_720p_video_off_cmd_1, 0 },
+};
+
+static struct command_state hx8394d_720p_video_state = {
+	.oncommand_state = 0,
+	.offcommand_state = 1,
+};
+
+static struct commandpanel_info hx8394d_720p_video_command_panel = {
+	/* Unused, this is a video mode panel */
+};
+
+static struct videopanel_info hx8394d_720p_video_video_panel = {
+	.hsync_pulse = 1,
+	.hfp_power_mode = 0,
+	.hbp_power_mode = 0,
+	.hsa_power_mode = 0,
+	.bllp_eof_power_mode = 1,
+	.bllp_power_mode = 1,
+	.traffic_mode = 2,
+	/* This is bllp_eof_power_mode and bllp_power_mode combined */
+	.bllp_eof_power = 1 << 3 | 1 << 0,
+};
+
+static struct lane_configuration hx8394d_720p_video_lane_config = {
+	.dsi_lanes = 4,
+	.dsi_lanemap = 0,
+	.lane0_state = 1,
+	.lane1_state = 1,
+	.lane2_state = 1,
+	.lane3_state = 1,
+	.force_clk_lane_hs = 0,
+};
+
+static const uint32_t hx8394d_720p_video_timings[] = {
+	0x7b, 0x1a, 0x10, 0x00, 0x3c, 0x91, 0x1c, 0x1c, 0x15, 0x03, 0x04, 0x00
+};
+
+static struct panel_timing hx8394d_720p_video_timing_info = {
+	.tclk_post = 0x1f,
+	.tclk_pre = 0x2d,
+};
+
+static struct panel_reset_sequence hx8394d_720p_video_reset_seq = {
+	.pin_state = { 1, 0, 1 },
+	.sleep = { 20, 1, 20 },
+	.pin_direction = 2,
+};
+
+static struct backlight hx8394d_720p_video_backlight = {
+	.bl_interface_type = BL_DCS,
+	.bl_min_level = 1,
+	.bl_max_level = 225,
+};
+
+static inline void panel_hx8394d_720p_video_select(struct panel_struct *panel,
+						   struct msm_panel_info *pinfo,
+						   struct mdss_dsi_phy_ctrl *phy_db)
+{
+	panel->paneldata = &hx8394d_720p_video_panel_data;
+	panel->panelres = &hx8394d_720p_video_panel_res;
+	panel->color = &hx8394d_720p_video_color;
+	panel->videopanel = &hx8394d_720p_video_video_panel;
+	panel->commandpanel = &hx8394d_720p_video_command_panel;
+	panel->state = &hx8394d_720p_video_state;
+	panel->laneconfig = &hx8394d_720p_video_lane_config;
+	panel->paneltiminginfo = &hx8394d_720p_video_timing_info;
+	panel->panelresetseq = &hx8394d_720p_video_reset_seq;
+	panel->backlightinfo = &hx8394d_720p_video_backlight;
+	pinfo->mipi.panel_on_cmds = hx8394d_720p_video_on_command;
```

Comment:
```suggestion
	pinfo->mipi.panel_on_cmds = hx8394d_720p_video_on_command;
	pinfo->mipi.panel_off_cmds = hx8394d_720p_video_off_command;
```

Can you check if this doesn't break the display? Not sure why lmdpdg doesn't generate this...

Reply:
Link: msm8916-mainline/linux-mdss-dsi-panel-driver-generator#48